### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by jesusvazquez**
Pull request to test in .net continuous integration server.

### CODE REVIEW

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by nuxcode** [comment]
retest this please

**Comment by nuxcode** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

**Comment by jesusvazquez** [comment]
retest this please

File: config/environment.rb [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+module Rails
+ class Application
+   class Configuration
+
+     def database_configuration
+       require 'erb'
+       if ENV['RAILS_DATABASE_FILE']
+         db_config = YAML.load(File.read(File.join(Rails.root, 'config/' + ENV['RAILS_DATABASE_FILE'])))
+       else
+         db_config = YAML.load(File.read(File.join(Rails.root, 'config/database.yml')))
```
Comment (rafatower) [inline config/environment.rb]: To be tested in staging. For the record, the original code:
```ruby
# file railties-3.2.20/lib/rails/application/configuration.rb +113

      # Loads and returns the contents of the #database_configuration_file. The
      # contents of the file are processed via ERB before being sent through
      # YAML::load.
      def database_configuration
        require 'erb'
        YAML::load(ERB.new(IO.read(paths["config/database"].first)).result)
      end
```

File: config/initializers/warden.rb [hunk 2]
```diff
@@ -175,21 +175,3 @@ def authenticate!
...
-    user_creation = Carto::UserCreation.where(user_id: user.id).first
-    return fail! unless user_creation
-
-    if user_creation.autologin?
-      success!(user, :message => "Success")
-    else
-      fail!
-    end
-  end
-
```
Comment (rafatower) [inline config/initializers/warden.rb]: review this, cause at first sight it shouldn't be deleted.

File: generateSpecFull.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Jesus Vazquez
+truncate -s 0 specfull.txt
+cat Makefile.parallel| grep -v '# S' | grep -v '# Pending' | grep 'rb' | sed -e 's/^\s*//' -e '/^$/d' | \
```
Comment (rafatower) [inline generateSpecFull.sh]: minor detail: `grep -v -i` if you want it case-insensitive

File: script/executor.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  do
+    if [ ! -f config/$redisfile.lock ]; then
+ #     echo $redisfile;
+      break
+    fi
+  done
+}
+
+main() {
+    fecha=$(date)
```
Comment (rafatower) [inline script/executor.sh]: unused

File: script/generateSpecFull.sh [hunk 5]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (rafatower) [inline script/generateSpecFull.sh]: better all these scripts within a dir `script/ci`

**Comment by rafatower** [comment]
:+1: :dancer:

**Comment by jesusvazquez** [comment]
retest this please

**Pull request merged**